types.d.ts

```diff
@@ -1,5 +1,6 @@
 declare module "react-xmasonry" {
     export interface XMasonryProps {
+      children?: React.ReactNode;
```

Comment:
Well, the only children of XMasonry component should be XBlock. Can we enforce it in the type?

Reply:
Can't make it work in my project. Tried:
```
  type XBlockElement =
    | React.ReactElement<XBlockProps>
    | boolean
    | null
    | undefined;
  export interface XMasonryProps {
    children?: XBlockElement | Array<XBlockElement>;
```

Getting `Type 'Element[]' is not assignable to type 'XBlockElement'`. Seems that `ComponentClass` doesn't pass prop types to render function.

Maybe let's just merge it as is, so at least it's compatible.